### ISSUE TITLE
Provide an uninstall message

### ISSUE THREAD

**Issue by rlindner81** [SAP/cf-mta-plugin#18]
Running `cf uninstall-plugin MtaPlugin`

Results in an error:
```
Uninstalling plugin MtaPlugin...
Could not find command with name 'CLI-MESSAGE-UNINSTALL'
The plugin's uninstall method returned an unexpected error.
The plugin uninstall will proceed. Contact the plugin author if you need help.
exit status 1
FAILED
```

This also happens when you update the plugin, because it triggers an uninstall.

**Comment by mptsonev** [comment]
Thanks for reporting. This is fixed in version 2.0.3 with commit: https://github.com/SAP/cf-mta-plugin/commit/3096d7a4a63a5c7e005dbf1850b919d865acc6d7

**Comment by daniel-kullmann** [comment]
This also happens when you try to install the same plugin twice or try to update the plugin:

```
>cf install-plugin -f out/MtaExample/cfMtaPlugin/dest/cf-cli-mta-plugin.bin                                          1 ↵
Attention: Plugins are binaries written by potentially untrusted authors.
Install and use plugins at your own risk.
Plugin MtaPlugin 2.0.0 is already installed. Uninstalling existing plugin...
Could not find command with name 'CLI-MESSAGE-UNINSTALL'
exit status 1
FAILED
```
